pyproject.toml

```diff
@@ -96,8 +71,8 @@ ignore=".venv"
 no-docstring-rgx="^(_|test_|Meta$)"
 
 [tool.pylint.messages_control]
-# Line length is enforced by Black, so pylint doesn't need to check it.
-# Pylint and Black disagree about how to format multi-line arrays; Black wins.
+# Line length is enforced by ruff, so pylint doesn't need to check it.
+# Pylint and ruff disagree about how to format multi-line arrays; ruff wins.
```

Comment:
This line doesn't make sense to me, as I'm not seeing anything disabled in pylint for multi-line arrays. Maybe it's a historical leftover and safe to remove?